src/sentry_plugins/splunk/plugin.py

```diff
@@ -303,13 +243,19 @@ def forward_event(self, event, payload):
                 },
             )
 
-            if isinstance(exc, (ConnectTimeout, ReadTimeout)):
-                # If we get a ConnectTimeout or ReadTimeout we don't need to raise an error here.
+            if isinstance(
+                exc,
+                (
+                    ApiHostError,
+                    ApiTimeoutError,
+                ),
+            ):
+                # The above errors are already handled by the API client.
                 # Just log and return.
                 return False
 
-            if isinstance(exc, SplunkError) and exc.status_code == 403:
+            if isinstance(exc, ApiError) and exc.status_code == 403:
                 # 403s are not errors or actionable for us do not re-raise
                 return False
 
-            raise
+            raise exc
```

Comment:
@NisanthanNanthakumar Nit: you can just call `raise` here and it will automatically re-raise the original exception

Reply:
ah didn't know that!